patterns/cell-based-eks/1.cell1/README.md

```diff
@@ -0,0 +1,181 @@
+# Cell-Based Architecture for Amazon EKS
+
+This example shows how to provision a cell based Amazon EKS cluster.
+
+* Deploy EKS Cluster with one managed node group in a VPC and AZ
```

Comment:
what is the motivation for mixing Fargate, managed nodegroup, and Karpenter in this design?

Reply:
It was about showing how to use them in single AZ pattern. Removed the Fargate and using 1 managed node group + Karpenter now.